### ISSUE TITLE
Warning compile latest git with gcc.

### ISSUE THREAD

**Issue by hnomkeng** [rathena/rathena#938]
git hash : 92bf61c
gcc : 4.4.7

![gcc1](https://cloud.githubusercontent.com/assets/9112684/12694907/b75c76da-c76e-11e5-8f11-8e459925a34c.jpg)
![gcc2](https://cloud.githubusercontent.com/assets/9112684/12694909/bd154d18-c76e-11e5-9da3-6ba124a53bac.jpg)

**Comment by Playtester** [comment]
battle.c seems to be modified because line 4248 for me is:
```c	
	switch (skill_id) {
```
skill.c:10773 doesn't match with my code either:
```c
	if (inf&INF_ATTACK_SKILL ||
		(inf&INF_SELF_SKILL && inf2&INF2_NO_TARGET_SELF) //Combo skills
		) // Casted through combo.
		inf = BCT_ENEMY; //Offensive skill.
```

Can you post the code you have around those lines in your source code?


skill.c:2509 - Confirmed. That variable is used nowhere.
Seems it's some old code from Cydh that's no longer needed because of the recent skill update by @aleos89.

Added:
https://github.com/rathena/rathena/commit/1ea4655140dc5d15e4faf59264e7cc8bac649a40

Now unused:
https://github.com/rathena/rathena/commit/fc6800701b9380f0ba086af1c8931cb8a162947f

**Comment by aleos89** [comment]
Fixed in 79d6eb2.
